### PR TITLE
fix: Fix get locator performance bug

### PR DESCRIPTION
**Pull request by TheWaWaR**
When get header from main chain we can get it from snapshot

### CODE REVIEW

File: store/src/store.rs [hunk 1]
```diff
@@ -188,7 +188,7 @@ pub trait ChainStore<'a>: Send + Sync {
     }
 
     fn is_main_chain(&'a self, hash: &packed::Byte32) -> bool {
-        self.get_block_number(&hash).is_some()
+        self.get(COLUMN_INDEX, hash.as_slice()).is_some()
```
Comment (keroro520) [inline store/src/store.rs]: What is the difference between `self.get_block_number(&hash).is_some()` and `self.get(COLUMN_INDEX, hash.as_slice()).is_some()`?

Comment (zhangsoledad) [reply store/src/store.rs]: `get_block_number` will do deserialize which is redundant for this function

**Comment by zhangsoledad** [comment]
bors r=zhangsoledad,keroro520

**Comment by zhangsoledad** [comment]
bors r=quake,keroro520,zhangsoledad

**Comment by doitian** [comment]
bors r=quake,keroro520,zhangsoledad

**Comment by doitian** [comment]
bors retry

**Pull request merged**